rllib/policy/torch_policy.py

```diff
@@ -120,6 +125,17 @@ def __init__(
             get_batch_divisibility_req(self) if get_batch_divisibility_req \
             else 1
 
+    @override(Policy)
+    def get_view_requirements(self):
```

Comment:
For clarity, should we rename these to `get_training_view_requirements` and `get_inference_view_requirements`? Or perhaps policy view / model view? I suppose inference view is always a subset of training view?

Reply:
Yes, it's actually that simple:
Model.get_view_requirements() -> always for forward pass.
Policy.get_view_requirements() -> can add additional fields required for postprocessing and loss.
I'll fix these inconsistencies.